tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/vertx/ActiveMQVertxUnitTest.java

```diff
@@ -792,4 +737,36 @@ public void tearDown() throws Exception {
       Thread.currentThread().setContextClassLoader(contextClassLoader);
       super.tearDown();
    }
+
+   private CoreQueueConfiguration createCoreQueueConfiguration(String queueName) {
+      return new CoreQueueConfiguration().setAddress(queueName).setName(queueName);
+   }
+
+   private ConnectorServiceConfiguration createOutgoingConnectorServiceConfiguration(HashMap<String, Object> config,
+                                                                                     String name) {
+      return new ConnectorServiceConfiguration().setFactoryClassName(VertxOutgoingConnectorServiceFactory.class.getName()).setParams(config).setName(name);
+   }
+
+   private ConnectorServiceConfiguration createIncomingConnectorServiceConfiguration(HashMap<String, Object> config,
+                                                                                     String name) {
+      return new ConnectorServiceConfiguration().setFactoryClassName(VertxIncomingConnectorServiceFactory.class.getName()).setParams(config).setName(name);
+   }
+
+   private HashMap<String, Object> createIncomingConnectionConfig(String vertxAddress, String incomingQueue) {
+      HashMap<String, Object> config1 = new HashMap<>();
+      config1.put(VertxConstants.HOST, host);
+      config1.put(VertxConstants.PORT, port);
+      config1.put(VertxConstants.VERTX_ADDRESS, vertxAddress);
+      config1.put(VertxConstants.QUEUE_NAME, incomingQueue);
+      return config1;
+   }
+
+   private HashMap<String, Object> createOutgoingConnectionConfig(String queueName, String vertxAddress) {
```

Comment:
a best refactoring would be to use URIs...

I don't think it's a big deal to make that change here.. I'm mentioning this just in case you start doing a similar thing on other tests.


I have been actually thinking about removing these modules.